apps/dcos_net/test/dcos_net_mesos_listener_tests.erl

```diff
@@ -111,7 +112,28 @@ tcp_udp_test() ->
                 #{name => <<"foobar">>, protocol => udp,
                   port => 6417}
             ],
-            state => {running, true}
+            state => running,
+            healthy => true
+        }
+    }, dcos_net_mesos_listener:from_state(Data)).
+
+unhealthy_test() ->
```

Comment:
Would it make sense to add few more "negative" test cases? For instance:
1. state is preparing and healthy is true  -- it shouldn't happen but our code shouldn't break either
2. state = '_' and health_check undefined -- as it use to happen earlier. 
3. state = terminal and healthy is true

Reply:
1. We'd prefer to use a real mesos state, but not to generate random data

2. There are a bunch of tests for case 2.

3. it is a regular case.